actionsystemtest/lib/action_system_test.rb

```diff
@@ -0,0 +1,153 @@
+  # from <tt>ActionSystemTestCase</tt>. System tests use Capybara as a base and
+  # allow you to configure the settings through your <tt>system_test_helper.rb</tt>
+  # file that is generated with a new application or scaffold.
+  #
+  # Here is an example system test:
+  #
+  #   require 'system_test_helper'
+  #
+  #   class Users::CreateTest < ActionSystemTestCase
+  #     test "adding a new user" do
+  #       visit users_path
+  #       click_on 'New User'
+  #
+  #       fill_in 'Name', with: 'Arya'
+  #       click_on 'Create User'
+  #
+  #       assert_text 'Arya'
+  #     end
+  #   end
+  #
+  # When generating an application or scaffold a +system_test_helper.rb+ will also
+  # be generated containing the base class for system testing. This is where you
+  # can change the driver, add Capybara settings, and other configuration for
+  # your system tests.
+  #
+  #   require "test_helper"
+  #
+  #   class ActionSystemTestCase < ActionSystemTest::Base
+  #     teardown do
+  #       take_failed_screenshot
+  #       Capybara.reset_sessions!
+  #     end
+  #   end
+  #
+  # By default, <tt>ActionSystemTest</tt> is driven by the Selenium driver, with
+  # the Chrome browser, on port 21800, and a browser size of 1400x1400.
+  #
+  # Changing the driver configuration options are easy. Let's say you want to use
+  # port 3000, and the Firefox browserinstead. In your +system_test_helper.rb+
+  # file add the following:
+  #
+  #   require "test_helper"
+  #
+  #   class ActionSystemTestCase < ActionSystemTest::Base
+  #     driven_by :selenium, using: :firefox, on: 3000
+  #
+  #     teardown do
+  #       take_failed_screenshot
+  #       Capybara.reset_sessions!
+  #     end
+  #   end
+  #
+  # +driven_by+ has a requried argument for the driver name. The keyword
+  # arguments are +:using+ for the browser (not applicable for headless drivers),
+  # +:on+ for port (the server is always Puma), and +:screen_size+ to change
+  # the size of the screen when taking screenshots.
+  #
+  # To use a headless driver, like Poltergeist, update your Gemfile to use
+  # Poltergeist instead of Selenium and then declare the driver name in the
+  # +system_test_helper.rb+ file. In this case you would leave out the +:using+
+  # option because the driver is headless.
+  #
+  #   require "test_helper"
+  #   require "capybara/poltergeist"
+  #
+  #   class ActionSystemTestCase < ActionSystemTest::Base
+  #     driven_by :poltergeist
+  #
+  #     teardown do
+  #       take_failed_screenshot
+  #       Capybara.reset_sessions!
+  #     end
+  #   end
+  #
+  # Because <tt>ActionSystemTest</tt> is a shim between Capybara and Rails, any
+  # driver that is supported by Capybara is supported by Action System Test as
+  # long as you include the required gems and files.
+  include Capybara::DSL
+  include ActionSystemTest::TestHelpers::ScreenshotHelper
+
+  class Base < ActionDispatch::IntegrationTest
```

Comment:
Let's go with TestCase like we do in all the other frameworks, so we have ActionSystemTest::TestCase. Then, in system_test_helper.rb, we can use `class ApplicationSystemTestCase < ActionSystemTest::TestCase`.